cmd/calyptia/create_core_instance_gcp.go

```diff
@@ -138,11 +149,26 @@ func newCmdCreateCoreInstanceOnGCP(config *config, client gcp.Client) *cobra.Com
 	fs.StringVar(&sshUser, "ssh-user", "", "GCP SSH user to use for the instance. (default is no user)")
 	fs.StringVar(&sshKeyPath, "ssh-key", "", "SSH Key path to use for the instance. (default is no key)")
 	fs.StringVar(&network, "network", "default", "GCP Network")
-	fs.BoolVar(&useTestImages, "use-test-images", false, "Use GCP test images instead of released channel (only for testing/development).")
 
 	return cmd
 }
 
+func getGitHubToken() (string, error) {
+	token := os.Getenv("CALYPTIA_GITHUB_TOKEN")
+	if token != "" {
+		return token, nil
+	}
+	return "", fmt.Errorf("CALYPTIA_GITHUB_TOKEN environment variable is not set")
+}
+
+func getTestEnvVar() bool {
+	useTest := os.Getenv("CALYPTIA_TEST_IMAGES")
```

Comment:
Does this enable test images or specify the test images to use? I'd expect the latter or an `ENABLE` string in the name.